### PR TITLE
Workthrough

### PR DESCRIPTION
**Pull request by ddalthorp**
Necessary revisions to splits code (so that calcRate works with SS). 
Repaired data sets (work fine on my desktop).
New rudimentary vignette with command line functions for looking at splits (and highlighting a couple problems).

### CODE REVIEW

**Comment by DAPPERstats** [comment]
Thanks!
I'll work on getting this stuff merged in
Can you set up the data files as .RData, not .rda, files please?

**Comment by DAPPERstats** [comment]
thanks! also, just an fyi since this ate some time for me earlier, but the capitalization in the extension matters, `.RData`, not `.Rdata` like I often accidentally write!

File: R/splits_functions.R [hunk 1]
```diff
@@ -36,29 +36,35 @@
...
+  } else if (!is.null(data_SS)){
+    stop(
+      "In arg list, data_SS must be an SS object. Alternatively, user may ",
+      "provide 'days' and 'carcass_searches'."
+    )
   }
   if (is.vector(M)){
     M <- matrix(M, nrow = 1)
     Aj <- matrix(Aj, nrow = 1)
     searches <- matrix(searches, nrow = 1)
```
Comment (DAPPERstats) [inline R/splits_functions.R]: should this be 
`searches_carcass <- matrix(searches_carcass, nrow = 1)`?

File: R/splits_functions.R [hunk 2]
```diff
@@ -36,29 +36,35 @@
 #'
 #' @export
 #'
-calcRate <- function(M, Aj, data_SS = NULL, days = NULL, searches = NULL){
+calcRate <- function(M, Aj, days = NULL, searches_carcass = NULL, data_SS = NULL){
  if (!is.null(data_SS) && ("SS" %in% class(data_SS))){
     days <- data_SS$days
-    searches <- data_SS$searches
+    unit <- rownames(Aj)
+    searches_carcass <- array(0, dim = c(x, length(days)))
```
Comment (DAPPERstats) [inline R/splits_functions.R]: `x` is undefined... it's supposed to be the number of carcasses, right? so just `nrow(Aj)`?

**Comment by DAPPERstats** [comment]
ok everything here should be incorporated as of v 0.0.5.14 #180
note that the code for `summary.Mest` is in `auxiliary_code/add_to_functions.R` for the time being so it's not currently available in the main package

File: R/splits_functions.R (hunk 2 above)
Comment (ddalthorp) [reply R/splits_functions.R]: yup. good catch.

File: R/splits_functions.R (hunk 1 above)
Comment (ddalthorp) [reply R/splits_functions.R]: yup. good catch.

**Comment by ddalthorp** [comment]
cool!

and summary.Mest to auxilliary code is a good idea.

**Comment by DAPPERstats** [comment]
ok, so i think at this point, this PR can be closed, as I've gotten everything over to the master branch. go ahead and close it whenever you're ready (if you're done working on it)

**Comment by ddalthorp** [comment]
OK. I've moved on to another, more up-to-date branch.

**Pull request closed without merging**